integration/flink/app/build.gradle

```diff
@@ -0,0 +1,368 @@
+/*
+/* Copyright 2018-2023 contributors to the OpenLineage project
+/* SPDX-License-Identifier: Apache-2.0
+*/
+
+import org.apache.tools.ant.filters.ReplaceTokens
+import io.franzbecker.gradle.lombok.task.DelombokTask
+
+plugins {
+    id 'java'
```

Comment:
Applying the `java` plugin is unnecessary, as the `java-library` plugin already does that. This can be safely removed.

Reply:
Removed